packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -771,6 +844,36 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
     return _currentBottomSheet;
   }
 
+  // FAB API
+  AnimationController _fabMoveController;
+  FabPosition _previousFabPosition;
+  FabPosition _fabPosition;
+
+  void _moveFab(final FabPosition newPosition) {
+    void updatePosition() {
+      setState(() {
+        _previousFabPosition = _fabPosition;
+        _fabPosition = newPosition;
+        print(
+          'Moving fab from $_previousFabPosition to $_fabPosition'
+        );
```

Comment:
i presume you didn't intend to leave the print

Reply:
No, it was just for debugging.  I've removed all the prints.